tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { expect, test } from '@playwright/test';
+import AggregatesComponent from 'Pages/AggregatesComponent';
+import ChatComponent from 'Pages/ChatComponent';
+import DashboardPage from 'Pages/DashboardPage';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { qase } from 'playwright-qase-reporter';
+import { TriggerDataSet } from 'testData/testData.enum';
+
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    TriggerDataSet.TriggerScenario,
+    TriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(TriggerDataSet.UserMail, TriggerDataSet.UserPassword);
+});
+
+test(
+  qase(
+    43,
+    '[Trigger] Event list should show at least 1 trigger, medium or low warning event',
+  ),
+  async ({ page }) => {
+    const aggregates = new AggregatesComponent(page);
+    const dashboard = new DashboardPage(page);
+    const userState = new UserStateComponent(page);
+    const chat = new ChatComponent(page);
+
+    // Navigate to disaster type the data was mocked for
+    await dashboard.navigateToFloodDisasterType();
+    // Assertions
+    await userState.headerComponentIsVisible({
+      countryName: TriggerDataSet.CountryName,
+    });
+    await chat.chatColumnIsVisibleForTriggerState({
+      firstName: TriggerDataSet.firstName,
+      lastName: TriggerDataSet.lastName,
+    });
+    await chat.allChatButtonsArePresent();
+
+    // get the number of warning events and aggregated events
+    const warningCount = await chat.chatPredictionButtonsAreActive();
+    const aggregatesNumber = await aggregates.getNumberOfPredictedEvents();
+
+    // check if the number of warning events is equal to the number of aggregated events
+    expect(warningCount).toEqual(aggregatesNumber);
```

Comment:
I'm not really getting this. I'm thinking maybe you're not fully clear on the terminology of 'trigger' (event above trigger) vs 'warning' (event below trigger threshold) vs 'alert' (this groups trigger+warning). Let's talk in person about this.

Reply:
call this 'chatEventCount' vs 'aggregatesEventCount'

Reply:
Changed!